### PR TITLE
[PP-4] Add GitHub actions with test and deploy stages to heroku

### PR DESCRIPTION
**Pull request by Praddby**
Task: https://andrei-lysiuk.atlassian.net/browse/PP-4
Description: Add github actions with test and deploy stages to heroku.

Deploy: https://pointing-poker-app.herokuapp.com

### CODE REVIEW

**Review by andreikaptsiuh — APPROVED** [review]

File: tsconfig.json [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-    "resolveJsonModule": true,
-    "isolatedModules": true,
-    "noEmit": true,
-    "jsx": "react-jsx"
-  },
-  "include": [
-    "src"
-  ]
-}
+{
```
Comment (mrakofff) [inline tsconfig.json]: Why this file in changes?

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    name: Deploy to Heroku
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: akhileshns/heroku-deploy@v3.12.12
+        with:
+          heroku_api_key: ${{secrets.HEROKU_API_KEY}}
+          heroku_app_name: "pointing-poker-app"
+          heroku_email: "pradd.wow@gmail.com"
+          procfile: "web: npm start"
```
Comment (mrakofff) [inline .github/workflows/deploy.yml]: use `yarn` instead of npm

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  build:
+    name: Deploy to Heroku
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: akhileshns/heroku-deploy@v3.12.12
+        with:
+          heroku_api_key: ${{secrets.HEROKU_API_KEY}}
+          heroku_app_name: "pointing-poker-app"
+          heroku_email: "pradd.wow@gmail.com"
```
Comment (mrakofff) [inline .github/workflows/deploy.yml]: Is it usefull?

File: .github/workflows/deploy.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+jobs:
+  build:
+    name: Deploy to Heroku
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: akhileshns/heroku-deploy@v3.12.12
+        with:
+          heroku_api_key: ${{secrets.HEROKU_API_KEY}}
+          heroku_app_name: "pointing-poker-app"
```
Comment (mrakofff) [inline .github/workflows/deploy.yml]: Move all heroku variables to secrets

File: .github/workflows/deploy.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+name: Deploy for FE
+
+on:
+  push:
+    branches:
+      - develop
+
+    needs:
+      - Test for FE
```
Comment (mrakofff) [inline .github/workflows/deploy.yml]: Do we need this?

**Review by mrakofff — CHANGES_REQUESTED** [review]
Leave some comments

File: tsconfig.json (hunk 1 above)
Comment (Praddby) [reply tsconfig.json]: I don't know. It is my problem with CRLF :(

Comment (Praddby) [reply tsconfig.json]: I fix it

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (Praddby) [reply .github/workflows/deploy.yml]: I think, it's only for Heroku.
Fix it.

File: .github/workflows/deploy.yml (hunk 4 above)
Comment (Praddby) [reply .github/workflows/deploy.yml]: ok

File: .github/workflows/deploy.yml (hunk 5 above)
Comment (Praddby) [reply .github/workflows/deploy.yml]: delete

File: .github/workflows/deploy.yml (hunk 3 above)
Comment (Praddby) [reply .github/workflows/deploy.yml]: yes

**Review by mrakofff — APPROVED** [review]

**Pull request merged**